samtranslator/model/connector/connector.py

```diff
@@ -20,6 +21,11 @@
     ],
 )
 
+STANDARD_CONNECTOR = "StandardConnector"
+EMBEDDED_CONNECTOR = "EmbeddedConnector"
+SOURCE_CONNECTOR = "SourceConnector"
```

Comment:
`SourceConnector` is a little vague. Maybe try something like `MultiDestConnector`?

Reply:
Sounds good.